src/Microsoft.ML.Data/DataLoadSave/TransformWrapper.cs

```diff
@@ -29,23 +29,27 @@ internal sealed class TransformWrapper : ITransformer
         private readonly IDataView _xf;
         private readonly bool _allowSave;
         private readonly bool _isRowToRowMapper;
+        private readonly bool _useLastTransformOnly;
```

Comment:
>_useLastTransformOnly [](start = 30, length = 21)

So I think now this class is always instantiated with `true`, isn't it? We wouldn't need the other case.
Same for `_allowSave`, I think it is always `false`.